tests/artemis-test-support/src/main/java/org/apache/activemq/transport/netty/NettyTransport.java

```diff
@@ -38,6 +38,8 @@
 
    ByteBuf allocateSendBuffer(int size) throws IOException;
 
+   void sendVoidPromise(ByteBuf output) throws IOException;
```

Comment:
I'm trying to understand why we would need a send with and without promise.


It seems to be all we need is to change send to be with a void promise.

please advise?

Reply:
Probably MQTT or Stomp (i don't remember) are using the version that need a proper callback while AMQP doesn't need it